db/migrate/20170725153927_create_data_requests.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDataRequests < ActiveRecord::Migration[5.1]
+  def change
+    create_table :data_requests, id: :uuid do |t|
+      t.integer :user_id
```

Comment:
`:user_id` is going to be false most of the time for now, this field exists as future-proofing in case we want to let people ask for dumps for a particular user's classification activity